armotypes/vulnerabilitytypes.go

```diff
@@ -76,29 +77,31 @@ type VulnerabilitiesComponent struct {
 
 type ComponentPathInfo struct {
 	ContainerPathInfo
-	ImageTag      string `json:"imageTag"`
-	ImageHash     string `json:"imageHash"`
-	ContainerName string `json:"containerName"`
+	ImageTag   string `json:"imageTag"`
+	ImageHash  string `json:"imageHash"`
+	IsRelevant string `json:"isRelevant"`
 }
 
 type Vulnerability struct {
-	Name           string                     `json:"name"`
-	Severity       string                     `json:"severity"`
-	SeverityScore  int                        `json:"severityScore"`
-	IsRCE          bool                       `json:"isRCE"`
-	Links          []string                   `json:"links"`
-	Description    string                     `json:"description"`
-	Exploitable    string                     `json:"exploitable"`
-	ComponentsInfo []VulnerabilitiesComponent `json:"componentsInfo"`
-	IsFixable      bool                       `json:"isFixable"`
-	CvssInfo       CvssInfo                   `json:"cvssInfo"`
-	EpssInfo       EpssInfo                   `json:"epssInfo"`
-	CisaKevInfo    CisaKevInfo                `json:"cisaKevInfo"`
+	Name           string                   `json:"name"`
+	Severity       string                   `json:"severity"`
+	SeverityScore  int                      `json:"severityScore"`
+	Links          []string                 `json:"links"`
+	Description    string                   `json:"description"`
+	Exploitable    string                   `json:"exploitable"`
+	IsRelevant     string                   `json:"isRelevant"`
+	ComponentsInfo VulnerabilitiesComponent `json:"componentInfo"`
+	CvssInfo       CvssInfo                 `json:"cvssInfo"`
+	EpssInfo       EpssInfo                 `json:"epssInfo"`
+	CisaKevInfo    CisaKevInfo              `json:"cisaKevInfo"`
+	WorkloadsCount int                      `json:"workloadsCount"`
```

Comment:
do we need it? or it for other pages? iif we want to support other pages so also component is multiple

Reply:
it's for cve page, wdym multiple? each cve + component + version is a different row